.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### What did you do? (steps to reproduce)
```

Comment:
I would add a heading above here 'Summary' and at the bottom 'Environment Info/URLs'

Reply:
Is this something you would like to have here but not in the RMT or should we align the two?

Reply:
Ok, as agreed during the standup I'll merge it as is and we can iterate on it later on.